aiortc/contrib/signaling.py

```diff
@@ -176,15 +177,44 @@ def client_connected(reader, writer):
         data = object_to_string(descr).encode('utf8')
         self._writer.write(data + b'\n')
 
+class WebsocketSignaling:
+    def __init__(self, host, port):
+        self._host = host
+        self._port = port
+        self._websocket = None
+
+    async def connect(self):
+        self._websocket = await websockets.connect("ws://" + str(self._host) + ":" + str(self._port))
```

Comment:
This means we don't support wss, I'm starting to wonder whether using urls for the signaling might be a good idea.

Reply:
this PR supports no encrypted websocket only ok?
or you say that modification which adds support of wss with --signaling-url option?

Reply:
You can keep it like this for now, it was just a comment for future me :)